src/plugins/intel_gpu/tests/CMakeLists.txt

```diff
@@ -47,7 +47,9 @@ target_link_libraries(${TARGET_NAME} PRIVATE openvino_intel_gpu_graph
 target_include_directories(${TARGET_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
                                                   ${CMAKE_CURRENT_SOURCE_DIR}/test_utils/
                                                   $<TARGET_PROPERTY:openvino_intel_gpu_kernels,INTERFACE_INCLUDE_DIRECTORIES>
-                                                  $<TARGET_PROPERTY:openvino_intel_gpu_runtime,INTERFACE_INCLUDE_DIRECTORIES>)
+                                                  $<TARGET_PROPERTY:openvino_intel_gpu_runtime,INTERFACE_INCLUDE_DIRECTORIES>
+                                                  ${CMAKE_HOME_DIRECTORY}/src/core/reference/include/
+                                                  ${CMAKE_HOME_DIRECTORY}/src/core/include/openvino/core/)
```

Comment:
do we need openvino/core?

Reply:
It was included to use shape.hpp, which I thought I needed in the developing process of TEST_P. As it is not needed anymore, it will be erased